addons/point_of_sale/views/pos_templates.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+<template id="pos_assets_backend" name="POS UI Backend Assets" inherit_id="web.assets_backend" primary="True">
+    <xpath expr="//script[contains(@src, '/mail/static/src/js/services/mail_service.js')]" position="replace"></xpath>
+    <xpath expr="//script[contains(@src, '/mail_bot/static/src/js/mailbot_service.js')]" position="replace"></xpath>
```

Comment:
will this crash if mail_bot is not installed?

Reply:
yep, it crashes

Reply:
It is in auto_install with only a depends on mail, so it should always be installed. But nothing prevents from uninstalling it. So a bridge module?

Reply:
or, new idea : how about disabling the chatter on the js side (not that I know how to do so)

Reply:
@alexkuhn is it possible to do what Martin suggested just above? i.e. disable clippy without removing it from the assets?

Reply:
@mart-e @len-odoo We removed these files from assets_backend and create new assets bundle and use it in POS UI template to not deploy mail service in pos UI, after checking I find that even if mail_bot service deployed it will not create any issue in pos UI but I removed mail service so that it is not deployed in pos UI otherwise if you have chat window open and if you load POS UI then those chat window will also be displayed POS UI.

Reply:
In v12, you cannot disable some services.
You'll need to add this feature on service provider mixin in order to ignore deployment of some services:
https://github.com/odoo/odoo/blob/12.0/addons/web/static/src/js/core/service_mixins.js#L57